docs/controls/Scatter.md

```diff
@@ -140,14 +140,15 @@ Refer [Graph](../core/Graph.md) `Root` for more details.
 
 #### Optional
 
-| Property Name | Expected | Default            | Description                                                           |
-| ------------- | -------- | ------------------ | --------------------------------------------------------------------- |
-| yAxis         | string   | "y"                | Setting for using different Y based axis. For now: its either Y or Y2 |
-| regions       | array    | []                 | Refer [Regions](#regions)                                             |
-| label         | object   | {}                 | Display value for the data-set which the data points belong to        |
-| color         | string   | COLORS.BLACK       | Color for the data point                                              |
-| shape         | string   | SHAPES.DARK.CIRCLE | Shape for representing the data points                                |
-| onClick       | Function | undefined          | Any action that can be performed when clicking on the data point      |
+| Property Name | Expected | Default            | Description                                                                       |
+| ------------- | -------- | ------------------ | --------------------------------------------------------------------------------- |
+| yAxis         | string   | "y"                | Setting for using different Y based axis. For now: its either Y or Y2             |
+| regions       | array    | []                 | Refer [Regions](#regions)                                                         |
+| label         | object   | {}                 | Display value for the data-set which the data points belong to                    |
+| color         | string   | COLORS.BLACK       | Color for the data point                                                          |
+| shape         | string   | SHAPES.DARK.CIRCLE | Shape for representing the data points                                            |
+| onClick       | Function | undefined          | Any action that can be performed when clicking on the data point                  |
+| legendOptions | object   | undefined          | Toggle to show legend. Refer [LegendOptions](#legendOptions)                      |
```

Comment:
Nit: properties are not alphabetized.

Reply:
Updated [here](https://github.com/NarasimhaShenoi/carbon-graphs/commit/1f13594bf10e008d1076eec74e3a34dd088c0384)